cognitedata/reveal
# feat(react-components): support for search of "second-level" 3d mapped instances

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-2630

## Description :pencil:

Adds suppport to search hooks that allows to search for "second-level" 3d mapped instances. Currently lacking support for direct relation mapped instances in allMappedEquipment hook but fully supported in the search hook.

## How has this been tested? :mag:


In SearchHooks story with work orders test data,

## Test instructions :information_source:


1. Go to SearchHooks story
2. Search for "work-order" and observe the results. 
3. You can check in Fusion Data models UI inside "FDX-Boys" data model that these views are only "indirectly" connected to 3d  model.


## Checklist :ballot_box_with_check:
